src/audio/module_adapter/library/userspace_proxy.c

```diff
@@ -74,7 +74,7 @@ static int userspace_proxy_memory_init(struct userspace_context *user,
 
 	struct k_mem_partition *parts_ptr[] = {
 #ifdef HEAP_PART_CACHED
```

Comment:
Can we align the Kconfig name in a subsequent PR?  It seems like a copy paste type issue.